src/styles/main.scss

```diff
@@ -1,3 +1,57 @@
+$cell-size: 100px;
+$flex-gap: 1px;
+$day-of-week: mon 0, tue 1, wed 2, thu 3, fri 4, sat 5, sun 6;
+
 body {
   margin: 0;
 }
+
+.calendar {
+    display: flex;
+    flex-wrap: wrap;
```

Comment:
I think its too much space before css rules, use 2 spaces for indentation

Reply:
lol how could i miss that 😁 sry